index.js

```diff
@@ -17,8 +18,15 @@ const newSessionHandlers = {
       this.attributes.gamesPlayed = 0;
       this.attributes.gameState = null;
     }
-    this.handler.state = states.STARTMODE;
-    this.emit(':ask', strings.welcome.output, strings.welcome.reprompt);
+    if (this.attributes.gamesPlayed === 0) {
```

Comment:
This comment is incorrect. Turns out there was just a bug that wasn't saving that info.